src/Http/Http.Extensions/test/RequestDelegateGenerator/Baselines/MapAction_ExplicitServiceParam_SimpleReturn_Snapshot.generated.txt

```diff
@@ -167,8 +167,15 @@ namespace Microsoft.AspNetCore.Http.Generated
                             httpContext.Response.StatusCode = 400;
                             return Task.CompletedTask;
                         }
-                        httpContext.Response.ContentType ??= "text/plain";
                         var result = handler(svc_local);
+                        if (result is string)
```

Comment:
Yeah, feels like we can do this in the same place where we write the response (as in the `ExecuteReturnAsync` handlers referenced above) instead of duplicating the check for hte result.

Reply:
I might be misunderstanding. If we don't have any filters we don't use `ExecuteObjectResult` unless I'm misreading the code.